### ISSUE TITLE
Dropout & K.learning_phase()

### ISSUE THREAD

**Issue by hccho2** [NLP-kr/tensorflow-ml-nlp#16]
어제 Dropout관련한 질문에 답변주셔서 감사합니다.

https://github.com/NLP-kr/tensorflow-ml-nlp/issues/15

추가로 한가지 더 질문드립니다.

```
if TRAIN:
	global_step = tf.train.get_global_step()
	train_op = tf.train.AdamOptimizer(0.001).minimize(loss, global_step)
	logging_hook = tf.train.LoggingTensorHook({"loss" : loss, "phase" : K.learning_phase() }, every_n_iter=400)
	return tf.estimator.EstimatorSpec(mode=mode, train_op=train_op, loss = loss,training_hooks = [logging_hook])

```

INFO:tensorflow:loss = 0.6956463, step = 1
INFO:tensorflow:loss = 0.6956463, phase = False

	
		
loggig_hook을 추가하여 K.learning_phase()를 찍어보면 False로 나옵니다.
K.learning_phase()이 train mode에서 자동으로 True가 되는게 맞나요?

**Comment by hccho2** [comment]
제가 이것 저것 test를 해보니, dropout은 정상적으로 적용되는 것 같습니다.
바깥에서 K.learning_phase() 찍었을 때는 False이지만, 내부에서 계산할 때는 일시적으로 True로 변하는게 아닌가 추정됩니다.
